script.js

```diff
@@ -1,12 +1,333 @@
 //You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+let allEpisodes; //= getAllEpisodes();
+let allShows; //= getAllShows();
+const rootElem = document.getElementById("root");
+rootElem.setAttribute("class", "root");
+let containerEl = document.getElementById("container");
+
+// all shows
+
+const loadShowList = (showID) => {
+  const urlShows = `https://api.tvmaze.com/shows/${showID}/episodes`;
+  fetch(urlShows)
+    .then((response) => {
+      if (response.status >= 200 && response.status <= 299) {
+        return response.json();
+      } else {
+        throw new Error(
+          `Encountered something unexpected: ${response.status} ${response.statusText}`
+        );
+      }
+    })
+    .then((episodeList) => {
+      allEpisodes = episodeList;
+
+      makePageForEpisodes(allEpisodes);
+    });
+};
+
+function displayAllShows(showList) {
+  showList.forEach((show) => {
+    let showContainer = document.createElement("section");
+    showContainer.className = "show-container";
+    containerEl.appendChild(showContainer);
+    // console.log(show);
+
+    // showname
+    const showName = document.createElement("h3");
+    showName.className = "show-name";
+    showName.innerHTML = `${show.name}`;
+    showName.addEventListener("click", (e) => {
+      let selectedName = e.target.innerHTML;
+      let selected = allShows.filter((show) => show.name === selectedName);
+      let showID = selected[0].id;
+      containerEl.innerHTML = "";
+      loadShowList(showID);
+      homeEl.style.display = "block";
+    });
+    showContainer.appendChild(showName);
+
+    // populating the select show
+    let optionShow = document.createElement("option");
+    optionShow.innerHTML = `${show.name}`;
+    selectShowEl.appendChild(optionShow);
+
+    // show thumbnails
+    const showImage = document.createElement("img");
+    if (show.image) showImage.src = `${show.image.medium}`;
+    else
+      showImage.src =
+        "https://img.freepik.com/premium-vector/error-404-landing-page-with-file-flat-design_249405-162.jpg?w=2000";
+    showContainer.appendChild(showImage);
+    // show ratings combo
+    const comboEl = document.createElement("section");
```

Comment:
Creating this `comboEl` container shows that you think of your web page's layout in advance.
Well done!